<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMachine</class>
 <widget class="QDialog" name="CreateMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="war1">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>30</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создание</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>179</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>432</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Состояние (Рабочий = 0, В ремонте = 1, Сломан = 2): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_state"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Состояние ремня (от 0 до 65535): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_belt_wear"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Состояние двигателя (от 0 до 65535): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_engine_wear"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Количество оборотов (от 0 до 32000): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_speed_engine"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="war2">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>67</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="war3">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>102</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="war4">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>138</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
